### PR TITLE
Add all shrines in Ratha to base-shrines.yaml

### PR DESCRIPTION
**Pull request by charleyc-dr**

### CODE REVIEW

File: data/base-shrines.yaml [hunk 1]
```diff
@@ -10,13 +10,6 @@ shrines:
     push: image
     path_out:
     - east
-  - location: The shrine of Tamsine's Rest
```
Comment (rpherbig) [inline data/base-shrines.yaml]: Why was this removed?

Comment (charleyc-dr) [reply data/base-shrines.yaml]: Duplicate; it's the same as lines 6-12.

Comment (rpherbig) [reply data/base-shrines.yaml]: 👌
